spiceaidocs/docs/components/data-connectors/file.md

```diff
@@ -75,6 +75,21 @@ datasets:
 
 When the file is modified, the acceleration will be refreshed and will include the latest data.
 
+## Types
```

Comment:
I suspect these need to be somewhere else instead of each `/data-connectors/*.md`. Otherwise, for each object store format, we will need to specify this conversion per format and per connector.

Reply:
Moved to refereces/datatypes/objectstore for common reference